### ISSUE TITLE
Asteroid Config XML

### ISSUE THREAD

**Issue by duncanwebb** [zmaster587/AdvancedRocketry#693]
Hi zmasterfox,

I've added the medium asteroid bit to asteroidConfig.xml and it looks pretty reasonable except the Iridium ore has the texture for Dilithium.

Is this a reasonable config?

```
<Asteroids>
	<asteroid name="Small Asteroid" distance="10" mass="100" massVariability="0.5" minLevel="0" probability="1" richness="0.2" richnessVariability="0.5">
		<ore itemStack="minecraft:iron_ore" chance="15" />
		<ore itemStack="minecraft:gold_ore" chance="10" />
		<ore itemStack="minecraft:redstone_ore" chance="10" />
	</asteroid>
	<asteroid name="Med Asteroid" distance="10" mass="500" massVariability="0.5" minLevel="0" probability="0.5" richness="0.2" richnessVariability="0.5">
	    <!-- ore0 is Dilithium, 8 is Rutile, 9 is Aluminium and 10 is Iridium -->
		<ore itemStack="libvulpes:ore0" chance="20" />
		<ore itemStack="libvulpes:ore0 8" chance="20" />
		<ore itemStack="libvulpes:ore0 9" chance="20" />
		<ore itemStack="libvulpes:ore0 10" chance="20" />
	</asteroid>
</Asteroids>
```

I looks good in the Observatory, I'll launch a rocket to mine it next.

![image](https://user-images.githubusercontent.com/4198031/26898925-d0ea36ac-4bcd-11e7-990f-90c427a78ba9.png)

**Comment by zmaster587** [comment]
I would personally make iridium have a lower chances so you get smaller quanities of it, but that's up to you if you want to do that or not.  
Also, item texture for iridium is now fixed.

**Comment by duncanwebb** [comment]
Thanks is was just a question to see I was doing this in the correctly, the balance is not reasonable and I think the distance should be larger, so it takes longer and need a better Observatory. Looks like the Observatory now collects data. And the texture is now correct.

Thanks a lot.

**Comment by zmaster587** [comment]
No problem

**Comment by ToserBan** [comment]
Hello zmasterfox,
I dont see Iridium Ore in the observatory. I tried the default asteroidConfig.xml and a similar one to that from duncanweb, i just deleted the comment line in the xml file. I play in mc 1.7.10 and use the latest alpha releases of advanced rocketry and libvulpes. But even in the last beta release i have the same problem. For example,  i find a medium asteroid or an enriched one, only the standart mc ores are displayed inside the obervatory gui. Where could be the problem ?

**Comment by zmaster587** [comment]
In the asteroid config file, make sure that the "V" in libVulpes is capitalized, if it's not then it will not properly load the iridium ore asteroid.

**Comment by ToserBan** [comment]
unfortunately, that didnt worked. Any other ideas ? Here is my config:
<Asteroids>
	<asteroid name="Small Asteroid" distance="10" mass="100" massVariability="0.5" minLevel="0" probability="10" richness="0.2" richnessVariability="0.5">
		<ore itemStack="minecraft:iron_ore" chance="15" />
		<ore itemStack="minecraft:gold_ore" chance="10" />
		<ore itemStack="minecraft:redstone_ore" chance="10" />
	</asteroid>
	<asteroid name="Iridium Enriched asteroid" distance="100" mass="25" massVariability="0.5" minLevel="0" probability="5" richness="0.2" richnessVariability="0.3">
		<ore itemStack="minecraft:iron_ore" chance="25" />
		<ore itemStack="libVulpes:ore0 10" chance="20" />
	</asteroid>
	<asteroid name="Med Asteroid" distance="10" mass="500" massVariability="0.5" minLevel="0" probability="5" richness="0.4" richnessVariability="0.5">
	    	<ore itemStack="minecraft:gold_ore" chance="40" />
	    	<ore itemStack="libVulpes:ore0 10" chance="40" />
	</asteroid>
</Asteroids

**Comment by ToserBan** [comment]
<Asteroids>
	<asteroid name="Small Asteroid" distance="10" mass="100" massVariability="0.5" minLevel="0" probability="10" richness="0.2" richnessVariability="0.5">
		<ore itemStack="minecraft:iron_ore" chance="15" />
		<ore itemStack="minecraft:gold_ore" chance="10" />
		<ore itemStack="minecraft:redstone_ore" chance="10" />
	</asteroid>
	<asteroid name="Iridium Enriched asteroid" distance="100" mass="25" massVariability="0.5" minLevel="0" probability="5" richness="0.2" richnessVariability="0.3">
		<ore itemStack="minecraft:iron_ore" chance="25" />
		<ore itemStack="libVulpes:ore0 10" chance="20" />
	</asteroid>
	<asteroid name="Med Asteroid" distance="10" mass="500" massVariability="0.5" minLevel="0" probability="5" richness="0.4" richnessVariability="0.5">
	    	<ore itemStack="minecraft:gold_ore" chance="40" />
	    	<ore itemStack="libVulpes:ore0 10" chance="40" />
	</asteroid>
</Asteroids>

**Comment by zmaster587** [comment]
Can you send it as a text or pastebin?  The config is only showing up as a white square for me

**Comment by zmaster587** [comment]
![image](https://user-images.githubusercontent.com/3536685/34920710-bf925a0e-f944-11e7-86c1-f306025f906f.png)

Using in the latest build of AR for 1.7.10
[asteroidConfig.txt](https://github.com/zmaster587/AdvancedRocketry/files/1629825/asteroidConfig.txt)

Iridium is not a garuntee, it's a probability, so it may take a few scans to find one.

**Comment by ToserBan** [comment]
if you reload this website  you will see the updated post with the screenshot. I tweaked the iridium and gold generation for testing, but only the gold appears in the observatory. I did about 30 scans with same result.
![grafik](https://user-images.githubusercontent.com/22052809/34921956-6f049e2c-f989-11e7-872d-8921b11b1f8c.png)

**Comment by ToserBan** [comment]
I found a solution although I dont know why I have to write the following that iridium will be shown in the observatory gui. I changed the asteroidconfig.xml as follows:
ore itemStack="libVulpes:libVulpesore0 10" chance="25"
instead of just
ore itemStack="libVulpes:ore0 10" chance="25"
For this solution i installed the mod bspkrsCore and started my world once and closed it. That mod then generated a file named UniqueNames.txt. In this file i saw, how minecraft named the libVulpes blocks...
Just strange, that it worked for you with the standard version of the asteroid config.
